docs/features.rst

```diff
@@ -729,4 +729,25 @@ the source should get pushed successfully from the ``force-app`` directory.
 Caveats:
 
 * It’s possible there may be some minor errors in the automatic conversion that need special attention in order to deploy successfully.
-* See https://ntotten.com/2018/05/11/convert-metadata-to-source-format-while-maintain-git-history/ for some tips on preserving git history while converting your source format.
+* See `this link <https://ntotten.com/2018/05/11/convert-metadata-to-source-format-while-maintain-git-history/>`_ for some tips on preserving git history while converting your source format.
+
+
+GitHub Gists
```

Comment:
Maybe we should give this a title like `Reporting error logs` that's more likely to be meaningful to the reader. Gists are a means to an end.